conclave-common/src/main/kotlin/com/r3/conclave/common/internal/attestation/QuoteVerifier.kt

```diff
@@ -469,7 +469,7 @@ object QuoteVerifier {
 
     private fun checkTcbLevel(pckExtensions: SGXExtensionASN1Parser, tcbInfo: TcbInfo): TcbStatus {
         val tcbLevelStatus = getMatchingTcbLevel(pckExtensions, tcbInfo)
-        check(tcbInfo.version == 2 || tcbLevelStatus != TcbStatus.OutOfDateConfigurationNeeded) {
+        check(tcbInfo.version == 2 || tcbInfo.version == 3 || tcbLevelStatus != TcbStatus.OutOfDateConfigurationNeeded) {
```

Comment:
I think the version check is redundant since it's checked during parsing. Probably another reason to use the enum in the tcbinfo object

Reply:
Removed the check, LMK if you think the method should go too.